### ISSUE TITLE
Feature request: "Don't ask me again" for keeping daemon running when closing GUI

### ISSUE THREAD

**Issue by Wegerich** [monero-project/monero-gui#2619]
I prefer to leave the GUI closed almost all the time while I am working on my computer. Occasionally I open it to check for payments, then close it again. It is unnecessary to prompt me about keeping the daemon open every time I close the software.

I would like to see an option in settings 

What should happen to the transaction monitoring node after the wallet GUI is closed?

- [x] Keep the node running so that my wallet is always ready to use **(recommended)**
- [ ] Always disable the node when closing. (Will require blockchain to synchronize when re-opened)
- [ ] Ask me what to do each time I close the wallet GUI

When the "ask me" prompt comes up, there could also be an option "don't ask me again" to check before selecting.

**Comment by rating89us** [comment]
I agree.
monerod should also be displayed as an icon in notification area of the taskbar.

**Comment by xmrdsc** [comment]
I don't think the GUI should be responsible for starting, stopping, and managing the daemon. It's should be just a client that can connect to a RPC port, whether that's on localhost or over the internet (remote node) is unimportant. So I would instead opt for a system service that manages the daemon with a taskbar icon and all.

**Comment by Wegerich** [comment]
> I don't think the GUI should be responsible for starting, stopping, and managing the daemon. It should only be a client that can connect to a RPC port, whether that's on localhost or over the internet (remote node) is unimportant. So I would instead opt for a system service that manages the daemon with a taskbar icon and all.

My proposal is a minor change to the current operation of the GUI. 

I feel that your suggestion is a major change to how the GUI interacts with the daemon and would be better suited to being its own separate GitHub issue.

**Comment by xmrdsc** [comment]
@Wegerich Yeah it's def. out of the scope for this issue, I agree that the prompt when closing is probably unneeded.

**Comment by ITwrx** [comment]
Yes, i always want my node closed when i close the GUI and find these questions and clicks fairly annoying. If a user hasn't initiated an action, they shouldn't be asked to work for the GUI, especially every time they want to use the GUI. It can feel like passive aggressive punishment instead of the helpful option it was intended to be. Like "oh, you wanted a GUI huh? well you won't mind if i ask you tons of questions then" :) Or "One doesn't simply make a GUI!"
